### PR TITLE
Cleaned zen-garden.html

### PR DESCRIPTION
**Pull request by phuongha2012**
Hi, I have cleared all the indentation in messy zen garden, please help me check this out!

### CODE REVIEW

File: pristine-zen-garden.html [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+            <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/" title="View the Creative Commons license of this site: Attribution-NonCommercial-ShareAlike." class="zen-license">CC</a>
+            <a href="http://mezzoblue.com/zengarden/faq/#aaa" title="Read about the accessibility of this site" class="zen-accessibility">A11y</a>
+            <a href="https://github.com/mezzoblue/csszengarden.com" title="Fork this site on Github" class="zen-github">GH</a>
+        </footer>
+
+    </div>
+
+
+    <aside class="sidebar" role="complementary">
+    <div class="wrapper">
```
Comment (guy-stapleton) [inline pristine-zen-garden.html]: After the aside, Indent div class="wrapper" and inner html tags

**Comment by guy-stapleton** [comment]
Hi Phuong, this looks good. The one thing I'ld suggest is indenting the div class="wrapper" and the inner html tags at lines 138-177 because divs and asides are both block level elements.

File: pristine-zen-garden.html (hunk 1 above)
Comment (phuongha2012) [reply pristine-zen-garden.html]: Hi Guy, thank you very much for your suggestion. I've made thisthis change to my pristine-zen-garden

**Comment by guy-stapleton** [comment]
That's great, I'll merge the request now

**Comment by phuongha2012** [comment]
Great thank you!

**Pull request merged**